signal_scanner_bot/env.py

```diff
@@ -32,7 +32,7 @@ def log_vars() -> None:
         log.debug(f"{key}={value}")
 
 
-DEBUG = _env("DEBUG", default=False)
+DEBUG = True if str(_env("DEBUG", default="false")).lower() == "true" else False
```

Comment:
This was set up so if you defined the variable to be anything it would be enabled. I'm not against trying to cast it, but maybe we should:
1. have a separate `_cast_to_bool` function and
2. add a `convert=` keyword to the `_env` function so we can pass that function in when defining these variables

E.g.
```
DEBUG = _env("DEBUG", default="false", convert=_cast_to_bool)
```

Reply:
Agreed updating this functionality to work as defined here and in subsequent comments.